src/systems/physics/Physics.cc

```diff
@@ -138,6 +139,7 @@ class ignition::gazebo::systems::PhysicsPrivate
           ignition::physics::GetEntities,
           ignition::physics::GetContactsFromLastStepFeature,
           ignition::physics::RemoveEntities,
+          ignition::physics::CollisionFilterMaskFeature,
```

Comment:
we made a few features optional in pull request #140 and I think we can also make the `CollisionFilterMaskFeature` optional as well. We need to merge 3 -> 4 forward first.

Reply:
merging 3 -> 4 in pull request #171

Reply:
ok PR merged. Can you merge with `master` and make this feature optional? thanks.

Reply:
Done, now it is optional, added to `CollisionFeatureList`